crates/api_models/src/payments.rs

```diff
@@ -489,6 +489,15 @@ pub struct Card {
     /// The card network for the card
     #[schema(value_type = Option<CardNetwork>, example = "Visa")]
     pub card_network: Option<api_enums::CardNetwork>,
+
+    #[schema(example = "CREDIT")]
+    pub card_type: Option<String>,
+
+    #[schema(example = "INDIA")]
+    pub card_issuing_country: Option<String>,
+
+    #[schema(example = "JP_AMEX")]
+    pub bank_code: Option<String>,
```

Comment:
It will be better if we bring as much of this data as we can under enums. Doesn't have to be in this PR. Please create an Issue for this.

Reply:
sure, will raise issue for others as well

Reply:
Please link the issue here once it is created